### PR TITLE
Version 0.6.12 ❄️ 

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/jmcantrell/python-dateutils)
[Jira Ticket
](https://anaconda.atlassian.net/browse/PKG-3562)
Channel: Snowflake

## Actions
- remove noarch
- update build parameters
- linter fixes

### CODE REVIEW

**Review by ryanskeith — COMMENTED** [review]
I am also seeing that this shouldn't work for any Python past 2.6.  Is this the right package we want to build?  This seems deprecated.

**Comment by cbouss** [comment]
> I am also seeing that this shouldn't work for any Python past 2.6. Is this the right package we want to build? This seems deprecated.

Where do you see this?

It has a wheel for py2-py3 on pypi: https://pypi.org/project/dateutils/0.6.12/#files

I don't see why pip check would fail though. It seems in order.

**Review by ryanskeith — APPROVED** [review]
Ah, you are right @cbouss .  I misread a line that said it needed to include argparse when python is less than 2.7. https://github.com/jmcantrell/python-dateutils/blob/d4be35229a0af59f18d0e754e76b91342eae02a4/setup.py#L38.

**Comment by SC426** [comment]
@cbouss You were right about pip check

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,40 +10,46 @@ source:
...
-    - python >=3
+  host:
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3
+    - python
     # - argparse only for py<27
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

**Review by lorepirri — APPROVED** [review]
LGTM, I appreciate the help and the link to the jira ticket!

**Review by cbouss — APPROVED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**